public/css/eventList.css

```diff
@@ -0,0 +1,80 @@
+.upper span {
+    display: block;
+    text-align: center;
+}
+
+.mid span {
+    display: block;
+    text-align: center;
+}
+
+#eventTitle {
+    text-align: center;
+}
+
+.event {
+    width: 80%;
+    overflow: auto;
+    margin: auto;
+    background-color: #fbd8ff;
+    margin-bottom: 10px;
+    border: solid #fdedff 2px;
+}
+
+.top {
+    width: 90%;
+}
+
+.upper, .mid {
+    width: 100%;
+    background-color: #fbd8ff;
+}
+
+.upper, .mid, .lower {
```

Comment:
Commas are bad, create a single class for all of them.

Reply:
Why?

Reply:
I tried to fix it but his HTML is organised with the assumption that you're styling with multiple commas. Fixing this would have involved entirely rewriting the html, so I settled for just seperating everything out into commas.

964193c